packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -106,21 +106,43 @@ function enableReviewWorkflow({ strapi }) {
       const { joinTable } = strapi.db.metadata.get(target).attributes[morphBy];
       const { idColumn, typeColumn } = joinTable.morphColumn;
 
-      // Execute a raw SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
+      // Execute an SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
       // Only entities that do not have a record in the join table yet are selected.
-      await strapi.db.connection
-        .raw(`INSERT INTO ${joinTable.name} (${idColumn.name}, field, "order", ${joinTable.joinColumn.name}, ${typeColumn.name})
-                SELECT
-                    entity.id as ${idColumn.name},
-                    '${ENTITY_STAGE_ATTRIBUTE}' as field,
-                    1 as "order",
-                    ${firstStage.id} as ${joinTable.joinColumn.name},
-                    '${contentTypeUID}' as ${typeColumn.name}
-                FROM ${contentTypeMetadata.tableName} entity
-                LEFT JOIN ${joinTable.name} jointable
-                ON  entity.id = jointable.${idColumn.name}
-                AND jointable.${typeColumn.name} = '${contentTypeUID}'
-                WHERE jointable.${idColumn.name} IS NULL`);
+      const connection = strapi.db.connection;
+      const columnsToInsert = [
+        idColumn.name,
+        'field',
+        `"order"`,
+        joinTable.joinColumn.name,
+        typeColumn.name,
+      ];
+
+      const selectStatement = connection
+        .select(
```

Comment:
we can discuss it tomorrow, but I think we should use strapi.db.getConnection() here, as it takes datbase schemas into consideration. (Dont ask me much what the are :D )

It also mentions in knex js that we should take the schema into consideration https://knexjs.org/guide/query-builder.html#identifier-syntax